tests/unit/test_main.cpp

```diff
@@ -1,2 +1,7 @@
 #define CATCH_CONFIG_MAIN
+#define CATCH_CONFIG_ENABLE_BENCHMARKING
```

Comment:
I have to disable benchmarks due to unspecified behavior in other tests. It's really confusing from chatch2 to cause SEGFAULT in totally unrelated tests which are solid, well-formed and correct.